sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/sas/TableAccountSasService.java

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables.sas;
+
+import com.azure.data.tables.implementation.StorageConstants;
+
+import java.util.Locale;
+
+/**
+ * This is a helper class to construct a string representing the services accessible by an Account SAS. Setting a value
+ * to true means that any SAS which uses these permissions will grant access to that service. Once all the values are
+ * set, this should be serialized with {@code toString()} and set as the services field on an
+ * {@link TableAccountSasSignatureValues} object. It is possible to construct the services string without this class, but
+ * the order of the services is particular and this class guarantees correctness.
+ */
+public final class TableAccountSasService {
```

Comment:
I think this class should be removed and just use `AccountSasService` from `azure-storage-common`.

Reply:
Please see my comment above.